libpod/networking_linux.go

```diff
@@ -249,9 +249,33 @@ func (r *Runtime) teardownNetNS(ctr *Container) error {
 	return nil
 }
 
+func getContainerNetNS(ctr *Container) (string, error) {
+	if ctr.state.NetNS != nil {
+		return ctr.state.NetNS.Path(), nil
+	}
+	if ctr.config.NetNsCtr != "" {
+		c, err := ctr.runtime.LookupContainer(ctr.config.NetNsCtr)
+		if err != nil {
+			return "", err
+		}
+		if err = c.syncContainer(); err != nil {
+			return "", err
+		}
+		return c.state.NetNS.Path(), nil
+	}
+	return "", nil
+}
+
 func getContainerNetIO(ctr *Container) (*netlink.LinkStatistics, error) {
```

Comment:
Maybe a comment here that nil, nil is a valid return if the container has no netns?

Reply:
fixed at return statement